.github/workflows/build.yml

```
@@ -0,0 +1,25 @@
+name: Build mod
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    name: Build mod
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '8'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: ./gradlew -Pnet.minecraftforge.gradle.disableUpdateChecker=true build
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: more-refined-storage
```

```suggestion
        name: morerefinedstorage.zip
```
Makes it a bit more clear what you get

---

Won't this make the filename `morerefinedstorage.zip.zip`? I haven't tested it, so I'll try, but that would be ugly.

---

Oh yeah you're right. I only looked at the artifact name but not at the downloaded file name in my testing.

---

Okay, I reverted it. This should be good to go now.